tap_sftp/client.py

```diff
@@ -26,6 +30,14 @@ def __init__(self, host, username, password=None, private_key_file=None, port=No
             key_path = os.path.expanduser(private_key_file)
             self.key = paramiko.RSAKey.from_private_key_file(key_path)
 
+        if timeout and float(timeout):
+            timeout_value = float(timeout)
+            # update the request timeout for the requests
+            self.request_timeout = timeout_value
+        else:
```

Comment:
Can you please add the comment related to 0, "0", "" values for else part.

Reply:
Added